### PR TITLE
DOCSP-19389 available method

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
## Pull Request Info

Added the `available()` method on the cursor page.

NOTE: API documentation link will need to be checked before publishing. (It currently doesn't work since it's not out)

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-19389

### Snooty build log:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=618be990c13055282dc5d7db

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/DOCSP-19389AvailableMethod/fundamentals/crud/read-operations/cursor/#number-of-results

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging and workerpool job links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [x] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [x] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

### CODE REVIEW

File: source/fundamentals/crud/read-operations/cursor.txt
Comment (ccho-mongodb) [inline source/fundamentals/crud/read-operations/cursor.txt:65]: Nitpick: it could be more clear if the explanation of the method "available" didn't include the word "available". Instead, consider using different words to describe it.

File: source/fundamentals/crud/read-operations/cursor.txt
Comment (ccho-mongodb) [inline source/fundamentals/crud/read-operations/cursor.txt:74]: Since this page is specific information about a Cursor (and there doesn't seem to be any other page that describes them), I wonder if it could be useful to describe what it means to "exhaust" a cursor. 

If the intent is for the reader to learn the term, maybe providing a definition could be helpful. Otherwise, you might try describing it without using the word "exhaust" since that's terminology we might not expect someone to know if they don't know much about cursors.

File: source/fundamentals/crud/read-operations/cursor.txt
Comment (ccho-mongodb) [inline source/fundamentals/crud/read-operations/cursor.txt:142]: I know you're planning to verify the link part, but I think this is misspelled.

```suggestion
- `available() <{+api+}/apidocs/mongodb-driver-sync/com/mongodb/client/MongoIterable.html#available()>`__
```

**Review by ccho-mongodb — APPROVED** [review]
LGTM after addressing a few issues. I think the resolutions are straightforward, so after fixing them and confirming the API link exists, feel free to advance this PR (whether external review or merge) without another review!

**Comment by kyuan-mongodb** [comment]
Hi @jyemin, This PR is ready for review. (It won't be merged in until I verify the API doc link works.)

**Review by jyemin — CHANGES_REQUESTED** [review]

**Review by jyemin — APPROVED** [review]

**Pull request merged**
